src/index.html

```diff
@@ -76,33 +76,33 @@ <h3 class="subtext">Input your stats</h3>
             <form class="sleep-form" action="index.html" method="post">
               <div class="input-container">
                 <label for="">Hours Slept</label>
-                <input class="hours-slept" type="number" step="any" min="0" name="" value="" placeholder="Enter Numerical Data" required>
+                <input class="hours-slept" type="number" step="any" min="0" name="hours-slept" value="" placeholder="Enter Numerical Data" required>
```

Comment:
All changes in index.html are just adding the class name as the name attribute value as well.